containerz/containerz.proto

```diff
@@ -257,6 +343,50 @@ message StartContainerRequest {
   // List of volumes that should be attached to the container.
   repeated Volume volumes = 7;
 
+  // The network to attach this container to. This could be 'host', 'bridged',
+  // or any other network available in the runtime.
+  string network = 8;
```

Comment:
Should this be an enumerated field, or is this entirely opaque to containerz?

Reply:
This is opaque to containerz and could be any string. While docker defines some network types, a user could deploy a network driver and have a different network type.

Reply:
SGTM.